src/models/CookieConsent.js

```diff
@@ -69,13 +61,25 @@ export default class CookieConsent extends Base {
   destroy(){
     return ( this.popup.destroy(), this )
   }
+  consents() {
+    return this.popup.consents;
+  }
 }
 
-statuses.reduce( ( obj, status ) =>
-( Object.defineProperty( CookieConsent, status, {
-  get: function () { return status },
-  set: function () {},
-  enumerable: false,
-  writeable: false,
-  configurable: false
-}), obj ), CookieConsent )
\ No newline at end of file
+STATUSES.reduce((obj, status) =>
```

Comment:
Please use plain `for (const status of statuses) { ... }` instead of `reduce` as this is not a pure function. Please apply the same for the ⬇️